drivers/snmp-ups.h

```diff
@@ -168,6 +171,32 @@ typedef struct {
 #endif /* WITH_SNMP_LKP_FUN */
 } info_lkp_t;
 
+/* Help align with DMF branch codebase until it is merged */
+#if WITH_SNMP_LKP_FUN
+# if (defined WITH_DMFMIB) && (WITH_DMFMIB != 0)
+#  define info_lkp_default(_1, _2)	{_1, _2, NULL, NULL, NULL, NULL}
+#  define info_lkp_fun_vp2s(_1, _2, _3)	{_1, _2, _3, NULL, NULL, NULL}
+#  define info_lkp_nuf_s2l(_1, _2, _3)	{_1, _2, NULL, _3, NULL, NULL}
+#  define info_lkp_fun_s2l(_1, _2, _3)	{_1, _2, NULL, NULL, _3, NULL}
+#  define info_lkp_nuf_vp2s(_1, _2, _3)	{_1, _2, NULL, NULL, NULL, _3}
+# else
+#  define info_lkp_default(_1, _2)	{_1, _2, NULL, NULL}
+#  define info_lkp_fun_vp2s(_1, _2, _3)	{_1, _2, _3, NULL}
+#  define info_lkp_nuf_s2l(_1, _2, _3)	{_1, _2, NULL, _3}
+/* no-op without DMF extensions */
+#  define info_lkp_fun_s2l(_1, _2, _3)	{_1, _2, NULL, NULL}
+#  define info_lkp_nuf_vp2s(_1, _2, _3)	{_1, _2, NULL, NULL}
+# endif /* WITH_DMFMIB */
```

Comment:
There is a mapping precedent in original eaton-marlin MIB, which uses the third pointer (so `info_lkp_fun_s2l`) in DMF branch, but the second pointer as the only one available in master. Not sure OTOH which one is correct; it could have been the case *why* the extra couple appeared in the first place.

Ideally, someone in the context should revise it more attentively, maybe test build of this branch against a device. @aquette - would you be up to this?

At the moment `info_lkp_fun_s2l` for non-DMF wrapper would ignore that function. Maybe they should re-use the first and second pointers, if the old (current master) codebase was so inclined?

Reply:
NOTE: Updated the macros to conflate the two `s2l` (string-to-long) and `vp2s` (void-pointer-to-string) macro names and so mapping table entries of the vanilla NUT, which became role-separated (forward and reverse `fun`/`nuf` conversions) in the DMF branch.